src/App.js

```diff
@@ -1,28 +1,33 @@
 import './styles/App.css';
 import images from './data.js'
 import CreatureList from './CreatureList.js'
-
-
-
-
-
-
-
-function App() {
-  return (
-    <>
-      <div className="App">
-        <header className="App-header">
-          <div>Welcome to Horned Beasts Gallery</div>
-        </header>
-      </div>
-      <div className="container">
-        <CreatureList images={images} />
-
-
-      </div>
-    </>
-  );
+import React, { Component } from 'react';
+
+export default class App extends Component {
+
+
+  state = {
+    filter: ''
+  }
+  handleChange = e => {
```

Comment:
are you passing this down as props, and how will it communicate with your CreatureList, then your filteredImage components, as well as state overall? Think this might be something you are missing.